src/api/nutrition.ts

```diff
@@ -1,54 +1,82 @@
+import { FoodListItem } from '@/components/modules/nutrition/add-food/food-list-item.model';
 
 import { dailyIntake } from '@/mocks/daily-intake';
 
 enum API_ENDPOINTS {
-    SEARCH = 'https://api.nutritionix.com/v2/search',
-    ITEM = 'https://api.nutritionix.com/v2/item'
+  SEARCH = 'https://api.nutritionix.com/v2/search',
+  ITEM = 'https://api.nutritionix.com/v2/item'
 }
 
 export class NutritionAPI {
-    private static localForageStore: LocalForage;
-
-    constructor() {
-        NutritionAPI.localForageStore = localForage.createInstance({name: 'nutritionDatabase'});
-        NutritionAPI.localForageStore.setItem('2019-08-13', dailyIntake('2019-08-13'));
-        NutritionAPI.localForageStore.setItem('2019-08-14', dailyIntake('2019-08-14'));
-    }
-
-    private get headers(): IHeaders {
-        return {
-            'x-app-id': appID,
-            'x-app-key': appKey,
-        };
-    }
-
-    @showLoader
-    public async getSearchItems(query: string): Promise<ISearchItem[]> {
-        const config: AxiosRequestConfig = {
-            params: {
-                q: query,
-            },
-            headers: this.headers,
-        };
-        return axios.get(API_ENDPOINTS.SEARCH, config).then((result: any) => result.data.results);
-    }
-
-    @showLoader
-    public async getItem(id: string): Promise<any> {
-        const config: AxiosRequestConfig = {
-            headers: this.headers,
-        };
-        return axios.get(`${API_ENDPOINTS.ITEM}/${id}`, config);
-    }
-
-    @showLoader
-    public async getMock(day: string): Promise<IDailyIntake> {
-        return NutritionAPI.localForageStore.getItem(day);
-    }
+  private static localForageStore: LocalForage;
+
+  constructor() {
+    NutritionAPI.localForageStore = localForage.createInstance({
+      name: 'nutritionDatabase',
+    });
+    NutritionAPI.localForageStore.setItem('2019-09-09', dailyIntake('2019-09-09'));
+  }
+
+  private get headers(): IHeaders {
+    return {
+      'x-app-id': appID,
+      'x-app-key': appKey,
+    };
+  }
+
+  @showLoader
+  public async getSearchItems(query: string): Promise<FoodListItem[]> {
+    const config: AxiosRequestConfig = {
+      params: {
+        q: query,
+        offset: 40,
+        limit: 40,
+      },
+      headers: this.headers,
+    };
+    return axios
+      .get(API_ENDPOINTS.SEARCH, config)
+      .then((result: any) => result.data.results);
+  }
+
+  @showLoader
+  public async getItem(id: string): Promise<FoodInfo> {
+    const config: AxiosRequestConfig = {
+      headers: this.headers,
+    };
+    return axios.get(`${API_ENDPOINTS.ITEM}/${id}`, config)
+      .then((v: any) => {
+        return {...v.data, resource_id: id};
+      });
+  }
+
+  @showLoader
+  public async getMock(day: string): Promise<IDailyIntake | null> {
+    return NutritionAPI.localForageStore.getItem(day);
+  }
+
+  public async updateCurrentDailyIntake(currentIntake: IDailyIntake, mealName: string, food: Food)
```

Comment:
I prefer to split method's params in separate lines and leave type with the closing `)`. I believe it's little more readable but that's only an opinion :)

Reply:
I tried to use this convention but it didn't convinced me